### DISCUSSION TITLE
License for jquery fancybox plugin in Prestashop 8

### DISCUSSION THREAD

**Discussion by TomaszFlo** [PrestaShop/PrestaShop#36216] [Q&A]
Hi,

In Prestashop 8, i find plugin /js/jquery/plugins/fancybox/jquery.fancybox.js
It is attached to the code at least in the backend panel on head section:
<script type="text/javascript" src="/js/jquery/plugins/fancybox/jquery.fancybox.js"></script>

Does the jquery fancybox plugin used in Prestashop 8 have a commercial license for the end store owner?

On the website of the owner of this plugin I read:
[https://github.com/fancyapps/fancyBox/issues/354]
"You can include fancyBox but must make sure each user understands that fancyBox is only free for non-commercial use.
Include a note in the description, something like -

This theme contains "fancyBox" by fancyApps (http://fancyapps.com/fancybox/).
Please note, that fancyBox is licensed under the therms of the Creative Commons Attribution-NonCommercial 3.0 License (http://creativecommons.org/licenses/by-nc/3.0/).
If you would like to use fancyBox for commercial purposes, you can purchase a license from http://fancyapps.com/store/"

**Comment by Progi1984** [comment]
May a new point to add to the EPIC about the legal compliance of the project : https://github.com/PrestaShop/PrestaShop/issues/22311
